.unreleased/PR_5930

```diff
@@ -0,0 +1,2 @@
+Fixes: #5930 Add update tests with weird user names for jobs
+Thanks: @mrksngl
```

Comment:
As far as I understand the changelog generation, the `<Thank you note>` seems to be missing here

Reply:
It was copied from #5613, but I am happy to add a note about this. Not sure how we handle duplicates in the tool.

Reply:
Done. PTAL.